code/clue.js

```diff
@@ -1,59 +1,267 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// Objects for suspects
 
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
   color: 'green',
-  description: 'He has a lot of connections',
-  age: 45,
+  description: 'He is connected.',
+  age: 'Age: ' + 45,
   image: 'assets/green.png',
-  occupation: 'Entrepreneur'
+  occupation: 'Occupation: entrepreneur',
+  favouriteWeapon: 'knife'
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'He is an old-school army man.',
+  age: 'Age: ' + 62,
```

Comment:
I would suggest strongly that these `age`s remain integer values and not strings.  
It will be harder to do math/numeric operations (greater/less than comparison, filtering etc.) if they are strings. 

You can always add a string label wherever you print them later